database/migrations/2020_05_13_141621_create_players_table.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreatePlayersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('players', function (Blueprint $table) {
+            $table->id();
+
+            $table->string('lk')->nullable();
+            $table->integer('dtb_id')->nullable();
```

Comment:
It's better to use full words for column names instead of abbreviations. You might have forgotten what 'lk' stands for in two months.